.circleci/config.yml

```diff
@@ -226,104 +288,154 @@ jobs:
+      - save_cache:
+          key: v5-pkg-cache-windows-1-15-{{ checksum "go.sum" }}
+          paths:
+            - C:\Users\circleci\AppData\Local\go-build
+            - C:\Users\circleci\go\pkg
+            - C:\Go\pkg
+  integration-deprecated-windows:
+    executor: win/default
+    environment:
+      OKTETO_CONTEXT: https://staging.okteto.dev
+      OKTETO_USER: cindylopez
+      OKTETO_APPS_SUBDOMAIN: staging.okteto.dev
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v5-pkg-cache-windows-1-15-{{ checksum "go.sum" }}
+      - attach_workspace:
+          at: .\artifacts
+      - run:
+          name: Install kubectl and helm
+          command: |
+            go version
+            choco install kubernetes-cli -y
+            choco install kubernetes-helm -y
       - run:
           name: Run deprecated integration tests
           environment:
-            OKTETO_URL: https://staging.okteto.dev/
             OKTETO_SKIP_CLEANUP: "true"
-            OKTETO_APPS_SUBDOMAIN: staging.okteto.dev
           command: |
             $env:OKTETO_PATH="$($HOME)\project\artifacts\bin\okteto.exe"
             $env:Path+=";$($HOME)\project\artifacts\bin"
+            & "$($HOME)\project\artifacts\bin\okteto.exe" analytics --disable
+            & "$($HOME)\project\artifacts\bin\okteto.exe" context use $env:OKTETO_CONTEXT --token $env:API_STAGING_TOKEN
             go test github.com/okteto/okteto/integration/deprecated/push -tags="integration" --count=1 -v -timeout 15m
             go test github.com/okteto/okteto/integration/deprecated/stack -tags="integration" --count=1 -v -timeout 15m
+
+  integration-build-windows:
+    executor: win/default
+    environment:
+      OKTETO_CONTEXT: https://staging.okteto.dev
+      OKTETO_USER: cindylopez
+      OKTETO_APPS_SUBDOMAIN: staging.okteto.dev
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v5-pkg-cache-windows-1-15-{{ checksum "go.sum" }}
+      - attach_workspace:
+          at: .\artifacts
       - run:
           name: Run build integration tests
           environment:
-            OKTETO_URL: https://staging.okteto.dev/
             OKTETO_SKIP_CLEANUP: "true"
-            OKTETO_APPS_SUBDOMAIN: staging.okteto.dev
           command: |
             $env:DEPOT_PROJECT_ID=$DEPOT_PROJECT_ID
             $env:DEPOT_TOKEN=$DEPOT_TOKEN
             $env:OKTETO_PATH="$($HOME)\project\artifacts\bin\okteto.exe"
+            $env:Path+=";$($HOME)\project\artifacts\bin"
+            & "$($HOME)\project\artifacts\bin\okteto.exe" analytics --disable
+            & "$($HOME)\project\artifacts\bin\okteto.exe" context use $env:OKTETO_CONTEXT --token $env:API_STAGING_TOKEN
             go test github.com/okteto/okteto/integration/build -tags="integration" --count=1 -v -timeout 10m
+
+  integration-deploy-windows:
```

Comment:
Can we rename them to `e2e` instead of `integration`? We started changing the name from integration to e2e but have to stop in the middle of the refactor